Assets/Scripts/Networking/GameManager.cs

```diff
@@ -63,7 +66,15 @@ public class GameManager : MonoBehaviourPunCallbacks
 
 		private void Awake()
 		{
+			GameObject conditionSetter = GameObject.Find("ConditionSetter");
+			if (isLiveTest && conditionSetter != null)
+            {
+				IsDDAEnabled = conditionSetter.GetComponent<ConditionSetter>().IsDDACondition();
+				Debug.LogError("condition string: " + conditionSetter.GetComponent<ConditionSetter>().condition);
```

Comment:
As I wrote above, using ConditionSetter as a GameObject and Monobehaviour seems a bit redundant